### PR TITLE
Essentials thermostat _TZE200_i48qyn9s

### PR DESCRIPTION
**Pull request by TAKeanice**
Reopen of #4909 (with adaptations, especially removing the schedule, that should make the PR acceptable). Resolves https://github.com/Koenkk/zigbee2mqtt/issues/14239

### CODE REVIEW

File: src/devices/essentials.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            .withFeature(e.numeric('minute', ea.STATE_SET).withUnit('min').withDescription('Start away minutes')),
+    ],
+    meta: {
+        tuyaDatapoints: [
+            [2, 'system_mode', tuya.valueConverterBasic.lookup({auto: tuya.enum(0), heat: tuya.enum(1), off: tuya.enum(2)})],
+            [16, 'current_heating_setpoint', tuya.valueConverterBasic.divideBy(2)],
+            [24, 'local_temperature', tuya.valueConverter.divideBy10],
+            [30, 'child_lock', tuya.valueConverter.lockUnlock],
+            [34, 'battery', essentialsValueConverter.battery],
+            [45, 'faultCode', essentialsValueConverter.faultCode],
```
Comment (Koenkk) [inline src/devices/essentials.ts:119]:
```suggestion
            [45, 'fault_code', essentialsValueConverter.faultCode],
```

File: src/devices/essentials.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            [101, 'comfort_temperature', tuya.valueConverter.divideBy10],
+            [102, 'eco_temperature', tuya.valueConverter.divideBy10],
+            [103, 'away_setting', essentialsValueConverter.away_setting],
+            [104, 'local_temperature_calibration', tuya.valueConverter.localTempCalibration1],
+            [105, 'schedule_override_setpoint', tuya.valueConverter.divideBy10],
+            [106, null, null], // TODO rapid heating
+            [107, 'window_open', tuya.valueConverterBasic.lookup({YES: true, NO: false})],
+            [108, null, null], // TODO hibernate
+            [116, 'open_window_temperature', tuya.valueConverterBasic.divideBy(2)],
+            [117, 'detectwindow_timeminute', tuya.valueConverterBasic.raw()],
```
Comment (Koenkk) [inline src/devices/essentials.ts:129]:
```suggestion
            [117, 'detect_window_timeminute', tuya.valueConverterBasic.raw()],
```

File: src/devices/essentials.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            output[7] = value.away_preset_days & 0xff;
+            output[6] = value.away_preset_days >> 8;
+            logger.debug(JSON.stringify({'send to tuya': output, 'value was': value}), NS);
+        },
+    },
+};
+
+const essentialsThermostat: Definition = {
+    fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_i48qyn9s'}],
+    model: 'Essentials Zigbee Radiator Thermostat',
```
Comment (Koenkk) [inline src/devices/essentials.ts:71]: Is there a better model number written on the device/product page?

Comment (TAKeanice) [reply src/devices/essentials.ts:71]: "essentials Heizkörperthermostat Zigbee Smart Home"

It's barely distinguishable from other non-Zigbee thermostats, and I haven't seen any significant model number or something like that on the packaging.

Comment (Koenkk) [reply src/devices/essentials.ts:71]: Then I would use `_TZE200_i48qyn9s` as the model.

Comment (TAKeanice) [reply src/devices/essentials.ts]: Done

**Comment by Koenkk** [comment]
Thanks!

Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `_TZE200_i48qyn9s.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**